docs/docs/overview.md

```diff
@@ -5,39 +5,47 @@ weight: 11
 
 ## ResyncPeriod
 
-Grafana doesn't have any webhooks or similar ways of giving information to the operator that a Grafana resource, like a dashboard, has changed.
-Due to this the Grafana operator has to constantly poll the Grafana API to test for changes in the dashboard.
+Grafana doesn't have any webhooks or similar ways of notifying the operator that a Grafana resource, like a dashboard, has changed.
+Due to this the Grafana operator constantly polls the Grafana API to test for changes and overwrite the resources, reconciling towards the desired state.
+
+We describe this loop as a synchronizing resources with Grafana instances.
 
 To control how often this polling should occur, you can set the `spec.resyncPeriod` field.
-This field tells the operator how often it should poll the Grafana instance for changes.
+This field tells sets how often it should poll the Grafana instance for changes for the specific resource.
```

Comment:
Two verbs instead of one ("tells sets"), I guess something like this would be fine: 

```suggestion
This field tells the operator how often it should poll the Grafana instance for changes for the specific resource.
```